compiler permission == PermissionStatus.disabled error

Compiler message:                                                       
../../flutter/.pub-cache/hosted/pub.dartlang.org/background_location-0.0.9+2/lib/background_location.dart:49:88: Error: Getter not found: 'disabled'.
    } else if (permission == PermissionStatus.denied || permission == PermissionStatus.disabled || permission == PermissionStatus.restricted || permission == PermissionStatus.unknown) {
                                                                                       ^^^^^^^^
Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
build failed.                                                           
                                                                        
FAILURE: Build failed with an exception.

--- a/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
+++ b/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
@@ -49,10 +49,7 @@ class LocationUpdatesService : Service() {
             val intent = Intent(this, LocationUpdatesService::class.java)
             intent.putExtra(EXTRA_STARTED_FROM_NOTIFICATION, true)
             
-            val activityPendingIntent = PendingIntent.getBroadcast(this, 0, Intent(STOP_SERVICE), 0)
             val builder = NotificationCompat.Builder(this)
-                    .addAction(R.drawable.abc_cab_background_top_material, "Stop service",
-                               activityPendingIntent)
                     .setContentTitle("Background service is running")
                     .setOngoing(true)
                     .setSound(null)

--- a/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
+++ b/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
@@ -47,10 +47,12 @@ class LocationUpdatesService : Service() {
     private val notification: Notification
         get() {
             val intent = Intent(this, LocationUpdatesService::class.java)
-
             intent.putExtra(EXTRA_STARTED_FROM_NOTIFICATION, true)
             
+            val activityPendingIntent = PendingIntent.getBroadcast(this, 0, Intent(STOP_SERVICE), 0)
             val builder = NotificationCompat.Builder(this)
+                    .addAction(R.drawable.abc_cab_background_top_material, "Stop service",
+                               activityPendingIntent)
                     .setContentTitle("Background service is running")
                     .setOngoing(true)
                     .setSound(null)

--- a/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
+++ b/android/src/main/kotlin/com/almoullim/background_location/LocationUpdatesService.kt
@@ -49,13 +49,9 @@ class LocationUpdatesService : Service() {
             val intent = Intent(this, LocationUpdatesService::class.java)
 
             intent.putExtra(EXTRA_STARTED_FROM_NOTIFICATION, true)
-
-            val activityPendingIntent = PendingIntent.getBroadcast(this, 0, Intent(STOP_SERVICE), 0)
-
+            
             val builder = NotificationCompat.Builder(this)
-                    .addAction(R.drawable.abc_cab_background_top_material, "Stop location Service",
-                            activityPendingIntent)
-                    .setContentTitle("Background Location Service is Running")
+                    .setContentTitle("Background service is running")
                     .setOngoing(true)
                     .setSound(null)
                     .setPriority(Notification.PRIORITY_HIGH)

--- a/lib/background_location.dart
+++ b/lib/background_location.dart
@@ -49,7 +49,6 @@ class BackgroundLocation {
         onGranted();
       }
     } else if (permission == PermissionStatus.denied ||
-        permission == PermissionStatus.disabled ||
         permission == PermissionStatus.restricted ||
         permission == PermissionStatus.unknown) {
       if (onDenied != null) {

